detection-rules/link_google_comment_script_macro.yml

```diff
@@ -0,0 +1,23 @@
+name: "Link to Google Apps Script macro via Comment Tagging"
+description: |
+  Message contains a Google Apps Script macro link invoked from a comment on Google Slides|Docs.
+  App Scripts can run arbitrary code, including redirecting the user to a malicious web page.
+references:
+  - "https://twitter.com/bunnymaid/status/1415478829162762240"
```

Comment:
if you have it handy, can we include a Playground link for future testing / reference?